mlir/test/Conversion/SCFToAffine/scf-affine-round-trip.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: imex-opt %s --scf-to-affine -lower-affine  -allow-unregistered-dialect -split-input-file| FileCheck %s
+
+// CHECK-LABLEL: copy_affine
+// CHECK: scf.parallel
+// CHECK-NEXT: memref.load
+// CHECK-NEXT: memref.store
```

Comment:
Presumably this is testing a conversion from scf to affine. The CHECK does not mention any affine, it just repeats the input. The input has only 2 scf operations and even one of them (parallel) seems to stay scf.
What is it testing?